Meta/CMake/flatpak/org.ladybird.Ladybird.json

```
@@ -0,0 +1,416 @@
+{
+  "app-id": "org.ladybird.Ladybird",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.9",
+  "sdk": "org.kde.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.llvm20"
+  ],
+  "command": "Ladybird",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
```

I'm still on X11 and had to add the fallback.

```suggestion
    "--socket=wayland",
    "--socket=fallback-x11",
```

---

I mean. I don't personally care about X11-only distros in $CURRENT_YEAR, but if it doesn't break anything I can add it back.